sdk/core/amqp-common/changelog.md

```diff
@@ -1,8 +1,8 @@
-
 ### 2019-7-3 1.0.0-preview.6
```

Comment:
we released the package yesterday, so do you want to keep the date as yesterday's only? @ramya-rao-a  @ramya0820